internal/managers/etc/manager.go

```diff
@@ -74,15 +73,9 @@ func (m *Manager) layoutInfo(g *gocui.Gui, x0, y0, x1, y1 int) error {
 				context = ""
 			}
 
-			if namespace, err = argoutil.GetNamespace(); err != nil {
-				m.log.Errorf("couldn't get the namespace: %s.", err)
-				namespace = ""
-			}
-
```

Comment:
it can make confuse to users because switching namespace works in Argocui, not context.